pkg/standalone-utils/contracts/test/ProtocolFeeBurnerMock.sol

```diff
@@ -11,9 +12,12 @@ import { FixedPoint } from "@balancer-labs/v3-solidity-utils/contracts/math/Fixe
 
 contract ProtocolFeeBurnerMock is IProtocolFeeBurner {
     using FixedPoint for uint256;
+    using SafeERC20 for IERC20;
 
     uint256 private _tokenRatio = FixedPoint.ONE;
 
+    bool tokenTakingEnabled = true;
+
     /// @inheritdoc IProtocolFeeBurner
     function burn(
```

Comment:
Do we want to rename `minTargetTokenAmountOut` to `minAmountOut` here as well, to match the rest of the renaming changes?